go.mod

```diff
@@ -78,9 +78,12 @@ require (
 
 require (
 	github.com/IGLOU-EU/go-wildcard v1.0.3
+	github.com/google/go-cmp v0.5.9
 	github.com/sigstore/k8s-manifest-sigstore v0.3.1-0.20220810053329-14f7cab4fd52
 )
 
+require github.com/weppos/publicsuffix-go v0.15.1-0.20220413065649-906f534b73a4 // indirect
```

Comment:
Where is this used?

Reply:
It was a dependency of `smallstep/zcrypto`, but now it's removed since I've switched back to using `x509` stdlib.